sql/src/main/java/io/crate/execution/dml/upsert/ShardUpsertRequest.java

```diff
@@ -313,7 +313,7 @@ public void writeTo(StreamOutput out, @Nullable Streamer[] insertValueStreamers)
                     Symbols.toStream(updateAssignment, out);
                 }
             } else {
-                out.writeVInt(0);
+                out.writeVInt(-1);
```

Comment:
If we've negative values using `writeInt` would be better

Reply:
Will change to a bool for this case. Then we can keep the vInt.